packages/owl-bot/test/commit-post-processor-update.ts

```diff
@@ -142,17 +146,19 @@ describe('commitPostProcessorUpdate', () => {
     const origin = makeOrigin();
     const clone = cloneRepo(origin);
     makeDirTree(clone, ['a.txt:The post processor ran.']);
-    const scope = nock('https://api.github.com')
-      .patch(`/repos/${destRepo}/pulls/${pr}`, {
-        draft: false,
-      })
-      .reply(204);
-    await commitPostProcessorUpdate(prepareArgs(clone));
+    assert.deepStrictEqual(
+      (await commitPostProcessorUpdate(prepareArgs(clone)))
+        .shouldPromoteFromDraft,
+      {
+        owner: 'test-org',
+        pull_number: pr,
+        repo: 'test-repo',
+      }
+    );
```

Comment:
This test reads strangely to mean, hence my nit about the name of the `.shouldPromoteFromDraft` name.

Reply:
done